background.js

```diff
@@ -226,6 +226,22 @@ const urls_to_param_mappers = [
     },
     {
         urls: ["*://getpocket.com/redirect*", "*://www.getpocket.com/redirect*"]
+    },
+    {
+        urls: ["*://t.mailpgn.com/l/*"],
+        query_param: 'fl'
+    },
+    {
+        urls: ["*://*.mailchimp.com/mctx/clicks*"]
+    },
+    // Leads to infinite redirect loop
```

Comment:
I wonder if the infinite redirect loop is caused by Medium using, say, the `Referer` header - would be a good thing to investigate and if so, an easy feature to work on. Maybe another time!